features/pull_request.feature

```diff
@@ -812,3 +812,36 @@ BODY
       Error creating pull request: Temporary Redirect (HTTP 307)
       Refused to follow redirect to https://disney.com/mouse\n
       """
+
+  Scenario: Default message with --push
+    Given the text editor adds:
+      """
+      """
+    Given the GitHub API server:
+      """
+      post('/repos/mislav/coral/pulls') {
+        halt 400 if request.content_charset != 'utf-8'
```

Comment:
You don't need this check. It's been put in another scenario to verify what the API client sends, but it doesn't need to be repeated more than once.